src/sass/Fabric.Icons.RTL.Output.scss

```diff
@@ -9,104 +9,349 @@
+.ms-Icon--CheckboxComposite,
+.ms-Icon--CheckMark,
+.ms-Icon--BackToWindow,
+.ms-Icon--FullScreen,
+.ms-Icon--Print,
+.ms-Icon--Save,
+.ms-Icon--EraseTool,
+.ms-Icon--PageLeft,
+.ms-Icon--PageRight,
+.ms-Icon--MultiSelect,
+.ms-Icon--Play,
+.ms-Icon--ChevronLeft,
+.ms-Icon--ChevronRight,
+.ms-Icon--ContactInfo,
+.ms-Icon--Unpin,
+.ms-Icon--WindowsLogo,
+.ms-Icon--Unlock,
+.ms-Icon--Calendar,
+.ms-Icon--Megaphone,
+.ms-Icon--AutoEnhanceOn,
+.ms-Icon--AutoEnhanceOff,
+.ms-Icon--Color,
+.ms-Icon--SaveAs,
+.ms-Icon--Light,
+.ms-Icon--Filters,
+.ms-Icon--Contrast,
+.ms-Icon--Redo,
+.ms-Icon--Undo,
+.ms-Icon--Album,
+.ms-Icon--Rotate,
+.ms-Icon--ThumbnailView,
+.ms-Icon--Package,
+.ms-Icon--Fiancial,
+.ms-Icon--ShoppingCart,
+.ms-Icon--Flag,
+.ms-Icon--Page,
+.ms-Icon--TouchPointer,
+.ms-Icon--TurnRight,
+.ms-Icon--Tab,
+.ms-Icon--Admin,
+.ms-Icon--EatDrink,
+.ms-Icon--SwitcherStartEnd,
+.ms-Icon--Touch,
+.ms-Icon--MapDirections,
+.ms-Icon--History,
+.ms-Icon--Recent,
+.ms-Icon--Hotel,
+.ms-Icon--News,
+.ms-Icon--Chat,
+.ms-Icon--Sync,
+.ms-Icon--Help,
+.ms-Icon--MailForward,
+.ms-Icon--ClosePane,
+.ms-Icon--OpenPane,
+.ms-Icon--PreviewLink,
+.ms-Icon--ZoomIn,
+.ms-Icon--Bookmarks,
+.ms-Icon--Document,
+.ms-Icon--ProtectedDocument,
+.ms-Icon--OpenInNewWindow,
+.ms-Icon--Switch,
+.ms-Icon--Rename,
+.ms-Icon--Folder,
+.ms-Icon--Picture,
+.ms-Icon--ShowResults,
+.ms-Icon--Message,
+.ms-Icon--MailReplyAll,
+.ms-Icon--PaymentCard,
+.ms-Icon--Copy,
+.ms-Icon--MailReply,
+.ms-Icon--Sort,
+.ms-Icon--Font,
+.ms-Icon--FolderFill,
+.ms-Icon--Permissions,
+.ms-Icon--DisableUpdates,
+.ms-Icon--Unfavorite,
+.ms-Icon--MoveToFolder,
+.ms-Icon--Dislike,
+.ms-Icon--Like,
+.ms-Icon--AlignRight,
+.ms-Icon--AlignCenter,
+.ms-Icon--AlignLeft,
+.ms-Icon--OpenFile,
+.ms-Icon--FontDecrease,
+.ms-Icon--FontIncraese,
+.ms-Icon--FontSize,
+.ms-Icon--Tag,
+.ms-Icon--Library,
+.ms-Icon--PostUpdate,
+.ms-Icon--NewFolder,
+.ms-Icon--CalendarReply,
+.ms-Icon--UnsyncFolder,
+.ms-Icon--SyncFolder,
+.ms-Icon--BlockContact,
+.ms-Icon--AddFriend,
+.ms-Icon--BulletedList,
+.ms-Icon--Preview,
+.ms-Icon--DockLeft,
+.ms-Icon--DockRight,
+.ms-Icon--Repair,
+.ms-Icon--Accounts,
+.ms-Icon--Stopwatch,
+.ms-Icon--Clock,
+.ms-Icon--AlarmClock,
+.ms-Icon--Timer,
+.ms-Icon--Annotation,
+.ms-Icon--Accept,
+.ms-Icon--Fingerprint,
+.ms-Icon--Handwriting,
+.ms-Icon--Stackindicator,
+.ms-Icon--Completed,
+.ms-Icon--Label,
+.ms-Icon--FlickLeft,
+.ms-Icon--FlickRight,
+.ms-Icon--MusicInCollection,
+.ms-Icon--OneDrive,
+.ms-Icon--CompassNW,
+.ms-Icon--LIghtningBolt,
+.ms-Icon--PrintfaxPrinterFile,
+.ms-Icon--Health,
+.ms-Icon--ChevronLeftSmall,
+.ms-Icon--ChevronRightSmall,
+.ms-Icon--ChevronLeftMed,
+.ms-Icon--ChevronRightMed,
+.ms-Icon--Dictionary,
+.ms-Icon--ChromeBack,
+.ms-Icon--PresenceChickletVideo,
+.ms-Icon--Reply,
+.ms-Icon--DoubleChevronLeftMed,
+.ms-Icon--CaretHollow,
+.ms-Icon--CaretSolid,
+.ms-Icon--Pinned,
+.ms-Icon--PinnedFill,
+.ms-Icon--Chart,
+.ms-Icon--BidiLtr,
+.ms-Icon--BidiRtl,
```

Comment:
Hmm, looks like BidiLtr and BidiRtl are directional icons that can't just be mirrored. So we may need to change the icon referenced by the class for these, instead of using CSS.